framework/packages/abstract-std/Cargo.toml

```diff
@@ -10,6 +10,7 @@ repository = "https://github.com/AbstractSDK/abstract"
 
 
 [features]
+test-utils = []
```

Comment:
Seems a bit overkill to add feature flag just for `TEST_ACCOUNT_ID`. We should be able to move that to `abstract-testing` package.